### ISSUE TITLE
fails to build lib/oxigraph on armel: no `AtomicU64` in `sync::atomic`

### ISSUE THREAD

**Issue by jonassmedegaard** [oxigraph/oxigraph#913]
Building v0.4.0-alpha.7 on Debian build-daemons fails on 32-bit architectures:
```
error[E0432]: unresolved import `std::sync::atomic::AtomicU64`
  --> lib/oxigraph/src/storage/memory.rs:16:25
   |
16 | use std::sync::atomic::{AtomicU64, Ordering};
   |                         ^^^^^^^^^
   |                         |
   |                         no `AtomicU64` in `sync::atomic`
   |                         help: a similar name exists in the module: `AtomicU32`
```

**Comment by jonassmedegaard** [comment]
Sorry, looking closer it really is only the i386 architecture that fails build like this.

**Comment by jonassmedegaard** [comment]
arrgh - not i386 but armel.

Full build log is [here](https://buildd.debian.org/status/fetch.php?pkg=oxigraph&arch=armel&ver=0.4.0%7Ealpha7-2&stamp=1720290005&raw=0)

**Comment by jonassmedegaard** [comment]
Perhaps relevant to use [atomic-polyfill](https://lib.rs/crates/atomic-polyfill)?

**Comment by adamreichold** [comment]
> Perhaps relevant to use [atomic-polyfill](https://lib.rs/crates/atomic-polyfill)?

I agree in principle, but I think the ecosystem is moving towards [`portable-atomic`](https://crates.io/crates/portable-atomic).

**Comment by Tpt** [comment]
Thank you! I think I'm going to move to `AtomicU32` on 32bits systems: allowing more than 2 billion transactions does not make much sense on these platforms.

**Comment by jonassmedegaard** [comment]
Applying #915 seems to solve this issue.

**Comment by Tpt** [comment]
Released as part of 0.4.0-alpha.8
